test/utils.test.js

```diff
@@ -0,0 +1,55 @@
+import { assembleCSV } from '../src/utils';
+import { expect } from 'chai';
+
+describe('utils', () => {
```

Comment:
Can we use `'utils.js'` here just to avoid any potential ambiguity?

Reply:
Fixed in 07c797ea6cedbe1e9583d113d3ccd73cd6b54fa8